CI "CMake build / macOS-Clang"
reports this error:

Error: /Users/runner/work/portaudio/portaudio/examples/paex_ocean_shore.c:395:58: error: implicit conversion from 'int' to 'float' changes value from 2147483647 to 2147483648 [-Werror,-Wimplicit-const-int-float-conversion]
  395 |     return minValue + ((maxValue - minValue) * rand()) / RAND_MAX;
      |                                                        ~ ^~~~~~~~

--- a/examples/paex_ocean_shore.c
+++ b/examples/paex_ocean_shore.c
@@ -387,9 +387,6 @@ OceanWave* InitializeWave(double SR, float attackInSeconds, float maxLevel, floa
     return wave;
 }
 
-/* This is failing CI because of "implicit conversion from 'int' to 'float'".
- * Make a new PR so we can reproduce the CI failure and fix it.
- */
 static float GenerateFloatRandom(float minValue, float maxValue)
 {
     return minValue + ((maxValue - minValue) * (float) rand()) / (float) RAND_MAX;

--- a/examples/paex_ocean_shore.c
+++ b/examples/paex_ocean_shore.c
@@ -392,7 +392,7 @@ OceanWave* InitializeWave(double SR, float attackInSeconds, float maxLevel, floa
  */
 static float GenerateFloatRandom(float minValue, float maxValue)
 {
-    return minValue + ((maxValue - minValue) * rand()) / RAND_MAX;
+    return minValue + ((maxValue - minValue) * (float) rand()) / (float) RAND_MAX;
 }
 
 /*******************************************************************/

